refactor/src/Cage.py

```diff
@@ -154,7 +152,6 @@ def update_lice_treatment_mortality(self, cur_date):
                                     (total_so_far + self.lice_population[stage])])
                     total_so_far += self.lice_population[stage]
                     if num_dead > 0:
-                        self.lice_population[stage] -= num_dead
```

Comment:
The population was being updated twice - once here and once in the update_deltas